src/requests/parameter_wrappers.rs

```diff
@@ -106,7 +106,19 @@ fn cql_value_from_napi_value(typ: &ComplexType, elem: &Array, pos: u32) -> napi:
             get_element!(BigInt),
             "Value cast into counter type shouldn't overflow i64",
         )?)),
-        CqlType::Decimal => todo!(),
+        CqlType::Decimal => {
+            let exp_size = 4;
+            let buf = get_element!(&[u8]);
+            // JS driver returns decimal in the same format as Decimal from CQL protocol.
+            // Returned buffer is in format XXXXYYYY...YYY
+            // where XXXX (4 bytes) is the exponent of the decimal in big endian
+            // and YYYY...YYY is the value of the decimal in big endian.
+            CqlValue::Decimal(CqlDecimal::from_signed_be_bytes_slice_and_exponent(
+                &buf[exp_size..],
+                i32::from_be_bytes(buf[0..exp_size].try_into().unwrap()), // Buffer is always guaranteed to be
+                                                                          // at least 4 bytes by JS part of the driver (it always returns at least 4 bytes).
```

Comment:
:pick: Please put the comment in the line above. The way it's now it's not readable.